packages/react-scripts/template/README.md

```diff
@@ -863,7 +863,20 @@ This feature is experimental and still [has major usage issues](https://github.c
 
 ## Deployment
 
-## Building for Relative Paths
+`npm run build` creates a `build` directory with a production build of your app. Set up your favourite http server so that a visitor to your site is served `index.html`, and requests to static paths like `/static/js/main.<hash>.js` is served with the contents of the `/static/js/main.<hash>.js` file.
+
```

Comment:
This jumps to an example too abruptly. Maybe mention "For example, you can serve static files with Python:"?
And then maybe add an Express example.